airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/sensors/metadata.py

```diff
@@ -20,26 +20,24 @@ def metadata_updated_sensor_definition(context: SensorEvaluationContext):
         with build_resources(resources_def) as resources:
             context.log.info("Got resources for gcs_metadata_updated_sensor")
 
-            etags_cursor_raw = context.cursor or None
-            etags_cursor = deserialize_composite_etags_cursor(etags_cursor_raw)
-            etags_cursor_set = set(etags_cursor)
+            etags_cursor_raw = context.cursor or []
+            etags_cursor = string_array_to_cursor(etags_cursor_raw)
```

Comment:
I'm not sure I get how this work. I thought cursor are MD5 strings, `etag_cursor_raw` is a string and not a list of string when `context.cursor` is not `False`, so `string_array_to_cursor` will be called with a string, not a string array.

Reply:
A damn, a hasty miss by me. Youre right this doesn't make sense.